readme.md

```diff
@@ -72,6 +72,20 @@ Default: `false`
 
 Format the number using the [Binary Prefix](https://en.wikipedia.org/wiki/Binary_prefix) instead of the [SI Prefix](https://en.wikipedia.org/wiki/SI_Prefix). This can be useful for presenting memory amounts. However, this should not be used for presenting file sizes.
 
+##### digits
+
+Type: `number`\
+Default: `3`
+
+Number of significant digits to display when calculating bytes or bits.
```

Comment:
I don't think "calculating" is the correct word to use here. And does it only apply when showing bytes and not megabytes?